docs/_includes/page-info-header.html

```diff
@@ -2,10 +2,10 @@
   assign wordWarning = 200 %} {% assign wordCount = content | number_of_words%} {%
   assign readTime = wordCount | divided_by: 100.0 | ceil %} {% if page.sub_title
   %}
-  <h1>{{ page.title }}</h1>
+  <h1>{{ page.name }}</h1>
```

Comment:
I couldn't work out how to reconcile the 'name' and 'title' of a repos collection in a way that shows the actual repo slug in the nav menu. The auto-generated titles are confusing, so have overridden the page.title to be the repo slug and page.name to be a friendly name for the title in the actual page, but maybe this needs a different pattern to be clear.

Reply:
The existing _repo collections md files will need